src/main/kotlin/no/nav/omsorgspenger/personopplysninger/RelasjonMediator.kt

```diff
@@ -0,0 +1,81 @@
+package no.nav.omsorgspenger.personopplysninger
+
+import kotlin.reflect.full.memberProperties
+import no.nav.omsorgspenger.personopplysninger.pdl.HentRelasjonPdlResponse
+import no.nav.omsorgspenger.personopplysninger.pdl.HentRelasjonPdlResponse.Relasjon
+import no.nav.omsorgspenger.personopplysninger.pdl.PdlClient
+import org.slf4j.LoggerFactory
+
+internal class RelasjonMediator(
+        internal val pdlClient: PdlClient
+) {
+    private val secureLogger = LoggerFactory.getLogger("tjenestekall")
+
+    suspend fun hentRelasjoner(identitetsnummer: String, til: Set<String>, correlationId: String): RelasjonsLøsningsMap {
+        val response = pdlClient.HentRelasjonInfo(til.plus(identitetsnummer), correlationId)
+
+        if (!response.errors.isNullOrEmpty()) {
+            secureLogger.error("Fann feil vid hent av data fra PDL: ", response.errors.toString())
+            throw IllegalStateException("Fann feil vid hent av data fra PDL")
+        }
+
+        var relasjonsListe = mutableListOf<Map<String, Any>>()
+
+        val søkersMatrikkelId = response.data.hentPersonBolk
```

Comment:
Vi må nok da grunnet Dolly hente både matrikkelId og gateadresse.
Der bør det nok lages en klasse, som f.eks. under som man kan lage tester rett på den for å verifisere den biten. Tror det blir veldig rotete med bare Strings. Også kan man jo da gjøre `søkersAdresse == annensAdresse` for å sette denne `borSammen`

```kotlin
    private data class Adresse(
        private val bostedMatrikkelId: String?,
        private val bostedGateadresse: String?,
        private val deltBostedMatrikkelId: String?,
        private val deltBostedGateadresse: String?) {
        override fun equals(other: Any?) = when {
            other !is Adresse -> false
            erLike(bostedMatrikkelId, other.bostedMatrikkelId) -> true
            erLike(bostedMatrikkelId, other.deltBostedMatrikkelId) -> true
            erLike(bostedGateadresse, other.bostedGateadresse) -> true
            erLike(bostedGateadresse, other.deltBostedGateadresse) -> true
            else -> false
        }

        private fun erLike(a: String?, b: String?) : Boolean {
            val beggeSatt = (a != null && a.isNotBlank()) && (b != null && b.isNotBlank())
            return beggeSatt && (a == b)
        }
    }
```